### PR TITLE
Feat/checkout/connect

### PR DESCRIPTION
**Pull request by abianco3**

### CODE REVIEW

File: server/controllers/artist.js [hunk 1]
```diff
@@ -22,7 +24,18 @@ router.get('/:artistId', (req, res) => {
   .catch(err => {
     res.status(400).send('Get artist info error!');
   });
+});
 
+router.post('/profile', authenticate, (req, res) => {
+  const profile = req.body.profile;
+  const artistId = req.user.userId;
+  model.profile(artistId, profile)
```
Comment (alisonzthu) [inline server/controllers/artist.js]: modle.profile? Don't see a function called profile in queries.js. Sure this work?

File: server/controllers/auctions.js [hunk 2]
```diff
@@ -36,9 +36,9 @@ router.post('/', authenticate, (req, res) => {
     res.status(500).send(serverErr);
   });
 });
-// SEEMS LIKE NO ONE IS POSTING TO THIS ENDPOINT
-router.post('/ongoing', (req, res) => {
-  const { user } = req.body;
+
+router.get('/ongoing', authenticate, (req, res) => {
```
Comment (alisonzthu) [inline server/controllers/auctions.js]: how come we need to authenticate this GET route?

File: server/controllers/artist.js (hunk 1 above)
Comment (abianco3) [inline server/controllers/artist.js]: I changed this to use the right query.

File: server/controllers/auctions.js (hunk 2 above)
Comment (abianco3) [inline server/controllers/auctions.js]: because they are the auctions that a individual user is bidding on, has won, etc.

**Pull request closed without merging**